tests/gnrc_dhcpv6_client/dhcpv6_server.sh

```diff
@@ -25,26 +25,25 @@ _kea_version_lesser_1_7_10() {
 
 _dhcpv6_server() {
     TMPDIR="$(mktemp -u | xargs dirname)/${APPLICATION}_kea"
+    CONFIG="${TMPDIR}/kea-dhcp6.config"
 
     # only used `mktemp` with dry-run above to get temp directory name, so we
     # still need to create the directory
     mkdir -p "${TMPDIR}"
-    if _kea_version_lesser_1_7_10; then
-        config=$2
-    else
+    sed "s/\"tapbr0\"/\"${IFACE}\"/" "$2" > "${CONFIG}"
```

Comment:
I think think this makes the conf file a template -> i think this should be more visible to the person looking at it
```suggestion
    sed "s/\"--IFACE--\"/\"${IFACE}\"/" "$2" > "${CONFIG}"
```
and since python is used for one case maybe it would be good for both and have it in a script instead of inline

Reply:
> and since python is used for one case maybe it would be good for both and have it in a script instead of inline

I think that would be overkill compared to a simple `sed`. With the logger config the actual structure of the JSON needs to be changed. Here we only want to search and replace multiple values.

Reply:
> I think think this makes the conf file a template -> i think this should be more visible to the person looking at it

Done, but used a syntax more common in other template languages, such as Jinja or Django.

Reply:
this may cause confusion in future, if someone looks at it an thinks there is  env.IFACE , lets use env.HOME for the memfile

Reply:
Then they may get confused, try it and notice that it does not work. But can add a comment, to alleviate potential confusion.

Reply:
> But can add a comment, to alleviate potential confusion.

Arghs, while Kea supports comments in their config, the JSON parser of python does not. So I added the doc on that to the test's README (and fixed it in the process)